data/controller_i18n_links.yaml

```diff
@@ -1,6 +1,14 @@
 # Translation links for Controller
 # Start Korean translations
 tower_ko:
+  v_4_4:
```

Comment:
```suggestion
  v_4_4_0:
```

Reply:
We are going to drop the _0 designation from now on out.